uint8 redValue
uint8 greenValue
uint8 blueValue
string ledAnimation
float64 flashFrequency